packages/theme-docs/src/i18n/en-US.js

```diff
@@ -6,6 +6,7 @@ export default {
     title: 'On this page'
   },
   article: {
-    github: 'Edit this page on GitHub'
+    github: 'Edit this page on GitHub',
+    updatedAt: 'Updated at'
```

Comment:
After this PR is merged, I will submit a PR to add Japanese translation for `updatedAt` text. 😄 

Or, if authors and collaborators want it to be included in one commit, please set:

```javascript
{
  updatedAt: '更新日'
}
```

Reply:
@NozomuIkuta I integrated your translation, thanks for it 👍